Remotion/Web/Development.WebTesting.IntegrationTests/Accessibility/AccessibilityIFrameSupportTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.Accessibility;
+using Remotion.Web.Development.WebTesting.ExecutionEngine.PageObjects;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.Accessibility
+{
+  [TestFixture]
+  public class AccessibilityIFrameSupportTest : IntegrationTest
+  {
+    [Test]
+    public void TestInjectSourceToIFrames ()
+    {
+      var home = Start<WxePageObject> ("Accessibility/IFrame.html");
+      var analyzer = Helper.CreateAccessibilityAnalyzer();
+
+      var accessibilityResult = home.Analyze (analyzer);
+      var violations = accessibilityResult.Violations.IgnoreByCssSelector ("html");
```

Comment:
am confused. also, wasn't there a not that ignore doesn't work for 'html'. am not 100% sure which exclude/filter that was exactly, but there was a note/guard somewhere.

Reply:
Ignore does work for `html`.

It was necessary here because the page containing the iframes had errors which had to be ignored. As the errors have been fixed, `.IgnoreByCssSelector ("html")` is obsolete now.

Reply:
okay, so we can remove it?

Reply:
Just to clarify my last comment:

`AccessibilityAnalyser.IgnoreCssSelector(...)` does not work with the `html` element (axe does not support it), this is also specified in its docs. The method that was used here was a filter method we're providing:

```csharp
public static IEnumerable<AccessibilityRuleResult> IgnoreByCssSelector (
    [NotNull] this IEnumerable<AccessibilityRuleResult> violations,
    [NotNull] params string[] cssSelector)
```